staff/edit-courses/get-dependents.php

```diff
@@ -0,0 +1,18 @@
+<?php
+    require_once "../../common.php";
+    require_staff();
+    header('Content-type: application/json');
+    $course_code = $_GET["course_code"];
+    unset($_GET['course_code']);
+    $course_id = $db->query("select course_id from course where course_code=?;", [$course_code])[0]["course_id"];
+    $dependent_ids = $db->query("SELECT course_id FROM requisite where dependent_id=?;", [$course_id]);
```

Comment:
This could be simplified a lot by using a join to get the course codes at the same time as the IDs